src/components/OcBase/Base.types.ts

```diff
@@ -0,0 +1,6 @@
+import React, { HTMLAttributes } from 'react';
+
+export interface OcBaseProps<T> extends HTMLAttributes<T> {
```

Comment:
@ychhabra-eightfold, come to think of it we may need to discuss this further and include @thiyagaraj-eightfold as he owns the task.

Reply:
yeah completely agree @dkilgore-eightfold, @thiyagaraj-eightfold please share ur proposal. 
Just to be mindfull of timelines i'm currently blocked on this, which is why I wanted to add data-test-id as a prop in every component.

Reply:
Hey -it's not really solving the problem of primitives, it's just creating a standard interface for the component. 
Just standardizing the props.
If it's blocking, let's merge it for now to unblock @ychhabra-eightfold

Reply:
Hey -it's not really solving the problem of primitives, it's just creating a standard interface for the component. 
Just standardizing the props.
If it's blocking, let's merge it for now to unblock @ychhabra-eightfold

Reply:
That was never the intention 🙂. I just wanted to add data-test-id everywhere, this was the only sane way of doing it imo. You can still work on creating primitives.